dockerclient.go

```diff
@@ -131,6 +131,11 @@ func (client *DockerClient) Info() (*Info, error) {
 }
 
 func (client *DockerClient) ListContainers(all bool, size bool, filters string) ([]Container, error) {
+	return client.ListContainersWithOptions(all, size, "", "", "", filters)
+}
+
+// ListContainersWithOptions behaves like ListContainers but allowing additionnal options
+func (client *DockerClient) ListContainersWithOptions(all, size bool, limit, since, before, filters string) ([]Container, error) {
```

Comment:
I think these options would be nicer as a struct, something like:

```golang
type ListContainerOptions struct {
    All, Size bool
    Limit, Since, Before, Filters string
}
```